.gitignore

```diff
@@ -1,5 +1,6 @@
 # Racket compiled files
 compiled/
+doc/
 
```

Comment:
Maybe leave this as a separate commit? (But what does this do?)

Reply:
When I installed the packages locally and built the docs, the CSS/JS/HTML appear in `pict-doc/pict/doc` as untracked files. AFAICT this is relatively standard for Racket packages, but I could be wrong.

This is a separate commit, but I left it in this PR.

I can remove it; the local equivalent is the same line in `.git/info/exclude`.

Reply:
I see. I think there are multiple ways one can get docs built; I'm using a git checkout with just "make" so I get pict that way (as it is part of main-distribution) and my docs are in `racket/doc`, which I see has a `.gitignore` already. So maybe this should be added?

Reply:
Now that I've taken Sam(th)'s advice and used `raco pkg update --clone`, I seem to no longer have this problem. I'll remove the commit unless you think it's worth keeping.